src/Provider/Provider.Web/Orchestrators/Reports/ProviderApplicationsReportOrchestrator.cs

```diff
@@ -49,23 +50,23 @@ public Task<Guid> PostCreateViewModelAsync(ProviderApplicationsReportCreateEditM
             switch (model.DateRange)
             {
                 case DateRangeType.Last7Days:
-                    fromDate = toDateInclusive.Date.AddDays(-7);
+                    fromDate = _timeProvider.Today.AddDays(-7);
```

Comment:
Can _timeProvider.Today be put into a variable and used in the other 3 locations?

Reply:
I'm not sure adding another variable into the mix simplifies this any. I think its descriptive enough.

Reply:
seems fine to me

Reply:
👍